backend/StreamServerApp/subtitles.py

```diff
@@ -66,4 +73,4 @@ def get_subtitles(video_path):
                     print(e.output)
                     raise
     print([webvtt_fr_fullpath, webvtt_en_fullpath])
-    return [webvtt_fr_fullpath, webvtt_en_fullpath]
+    return [webvtt_fr_fullpath, webvtt_en_fullpath, webvtt_ov_fullpath]
```

Comment:
I think it would be more appropriate to return a tuple instead of a list.
I don't think it has much impact, but it is always good to follow conventions.